### PR TITLE
PW-92: Add $lenmulti function to scripting documentation

### PR DESCRIPTION
**Pull request by rdswift**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [x] Other
* **Describe this change in 1-2 sentences**: This adds the documentation for the scripting `$lenmulti` function.

# Problem

At some point the $lenmulti() function was added to Picard, but it seems that it was not documented in the Scripting section of the website.

* JIRA ticket: [PW-92](https://tickets.metabrainz.org/browse/PW-92)

# Solution

Add the $lenmulti() function documentation to the Scripting section of the website.

# Action

### CODE REVIEW

File: website/frontend/templates/docs/scripting.html [hunk 1]
```diff
@@ -611,6 +611,16 @@ <h3 id="len_text">$len(text)</h3>
...
         </ul>
 
+        <h3 id="lenmulti_name_separator">$lenmulti(name,separator="; ")</h3>
+
+        <ul>
+          <li>Returns the number of elements in the multi-value tag
+            <code>name</code>. A literal value representing a multi-value can
+            be substituted for <code>name</code>, using the
+            <code>separator</code> (or "; " if not passed) to coerce the value
+            into a proper multi-valued tag.</li>
```
Comment (phw) [inline website/frontend/templates/docs/scripting.html]: I have the feeling we should have an example here :) I guess many will have a hard time understanding this

Comment (rdswift) [reply website/frontend/templates/docs/scripting.html]: Is this example okay, or should I include something like `$lenmulti(%artists%)` will return "3" for release https://musicbrainz.org/release/8c759d7a-2ade-4201-abc2-a2a7c1a6ad6c

Comment (phw) [reply website/frontend/templates/docs/scripting.html]: I think that works fine

**Comment by rdswift** [comment]
I just found a mistake in the description of the `$while` function, and need to delete two words.  I created a ticket [PW-93](https://tickets.metabrainz.org/browse/PW-93) for it.  Just wondering if I can add it to this PR or if you'd prefer I create a new PR for it?

**Comment by phw** [comment]
> Just wondering if I can add it to this PR or if you'd prefer I create a new PR for it?

Yes, let's keep it simple for this. Also opposed to Picard itself we also don't really need a ticket entry for every change on the website. So just a PR is fine for most changes. A ticket makes sense if you just want to report it or if there is some more need of coordination.

For Picard it is a bit different, as we use the tickets there for automatically getting the changelog. So the general rule is every change that is noteworthy in the changelog must have a ticket.

**Comment by rdswift** [comment]
I added the small correction here.  If I misunderstood, I can revert it and put up a new PR.

**Pull request merged**